### DISCUSSION TITLE
Еда

### DISCUSSION THREAD

**Discussion by registant** [mindryder/ATLAS-RU#5] [НАЙДЕННЫЙ ПРОПУЩЕННЫЙ ПЕРЕВОД]
![Screenshot_1338](https://user-images.githubusercontent.com/123338303/214250722-518a2a6a-f452-4965-9182-174642238e4f.jpg)
![Screenshot_1340](https://user-images.githubusercontent.com/123338303/214250729-9535939a-4d03-44e0-b4ac-9fec40eb6f5d.jpg)
![Screenshot_1339](https://user-images.githubusercontent.com/123338303/214250734-a48b2d02-45d8-4ef0-85a1-052459e6d3fe.jpg)
Нет возможности сейчас проверить всю еду, но вижу что некоторые вещи не переведены.

**Comment by mindryder** [comment]
Перевод предметов ещё продолжается
